src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -12,6 +12,10 @@ internal static partial class Interop
 {
     internal static partial class Ssl
     {
+        internal const int OPENSSL_NPN_NEGOTIATED = 1;
+        internal const int SSL_TLSEXT_ERR_OK = 0;
+        internal const int SSL_TLSEXT_ERR_NOACK = 3;
+
```

Comment:
I can't seem to seen native asserts for these, should there not be something like this

`static_assert(PAL_X509_V_ERR_CRL_SIGNATURE_FAILURE == X509_V_ERR_CRL_SIGNATURE_FAILURE, "");`

To ensure that the native and managed code never get out of sync?